wagtail/wagtailsnippets/templates/wagtailsnippets/snippets/confirm_delete.html

```diff
@@ -1,5 +1,5 @@
 {% extends "wagtailadmin/base.html" %}
-{% load wagtailadmin_tags i18n %}
+{% load wagtailadmin_tags i18n admin_urls %}
```

Comment:
We can't rely on user projects having `django.contrib.admin` in `INSTALLED_APPS`, and if it isn't, the tag library will fail to load. We should probably import or copy the `admin_urlquote` filter into wagtailadmin_tags instead...